test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -51,15 +60,35 @@ segmentItems:
             peer: ''
             peerId: 0
             tags:
-              - {key: url, value: not null}
+              - {key: url, value: 'http://localhost:8080/testcase/annotation/hello'}
+              - {key: http.method, value: GET}
+              - {key: status_code, value: '200'}
             refs:
-              - {parentEndpointId: 0, parentEndpoint: /projectA/testcase, networkAddressId: 0,
-                 entryEndpointId: 0, refType: CrossProcess, parentSpanId: not null, parentTraceSegmentId: not null,
-                 parentServiceInstanceId: not null, networkAddress: not null, entryEndpoint: /projectA/testcase,
+              - {parentEndpointId: 0, parentEndpoint: /testcase/annotation/hello, networkAddressId: 0,
+                 entryEndpointId: 0, refType: CrossProcess, parentSpanId: 0, parentTraceSegmentId: not null,
+                 parentServiceInstanceId: not null, networkAddress: 'localhost:8080', entryEndpoint: /testcase/annotation/hello,
                  entryServiceInstanceId: not null}
       - segmentId: not null
         spans:
-          - operationName: org.apache.skywalking.apm.testcase.sc.webflux.projectB.controller.TestAnnotationController.error
+          - operationName: /testcase/annotation/hello
+            operationId: 0
+            parentSpanId: -1
+            spanId: 0
```

Comment:
Why are all these spans id=0 and parentSpanId=-1, and no ref?  What is the relationship with other segments?